addons/binding/org.openhab.binding.enocean/ESH-INF/thing/serialbridge.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="enocean"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- EnOcean Bridge -->
+	<bridge-type id="serialbridge">
+		<label>EnOcean Serial Bridge</label>
```

Comment:
In the readme, you call those an "EnOcean Gateway" - what is the official naming?